tests/integration_tests/document_loaders/test_fauna.py

```diff
@@ -0,0 +1,36 @@
+import unittest
```

Comment:
1) Could you add `fauna` as an optional to the main group, and register `fauna` in the extended testing group
2) Add a unit test that uses @pytest.mark.parameterize and attempts to import the fauna loader (or even better uses a fixture to test the glue code between the fauna client and the results of the loader)

Reply:
updated. please review

Reply:
Hi @Shadid12!

Thanks for the unit test!

---

Please see instructions here on how to register in the optional dependency: 

https://github.com/hwchase17/langchain/blob/master/.github/CONTRIBUTING.md#working-with-optional-dependencies

The dependency needs to be added to the `extended_testing`  extra in pyproject.toml

---

langchain relies on the pytest framework for unit testing -- classes should not inherit from `unittest.TestCase` -- there might be a few exceptions to this in the code base, but most of the code is relying on pytest style testing